presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -101,4 +103,17 @@ public InternalCommunicationConfig setBinaryTransportEnabled(boolean binaryTrans
         this.binaryTransportEnabled = binaryTransportEnabled;
         return this;
     }
+
+    public DataSize getTaskUpdateSizeLimit()
+    {
+        return taskUpdateSizeLimit;
+    }
+
+    @Config("experimental.internal-communication.task-update-size-limit")
```

Comment:
Instead of saying "size-lmit", we usually say `experimental.internal-communication.max-task-update-size`